compiler/resolution/src/org/jetbrains/kotlin/resolve/calls/components/ResolutionParts.kt

```diff
@@ -597,6 +598,28 @@ internal object CheckReceivers : ResolutionPart() {
         resolveKotlinArgument(receiverArgument, receiverParameter, receiverInfo)
     }
 
+    private fun KotlinResolutionCandidate.searchForAdditionalReceivers() {
```

Comment:
I'd made this function returning nullable list of receivers
`null` in case no potential receiver found
For me it would look more clear and we won'y start calling `checkReceivers` while it's not needed anyway